cmake/catkin_install_python.cmake

```diff
@@ -40,7 +40,10 @@ function(catkin_install_python signature)
         file(MAKE_DIRECTORY ${rewritten_file})
         # even though the content of the file is overwritten
         # the copy makes sure the file has the right permissions
-        file(COPY ${source_file} DESTINATION ${rewritten_file} USE_SOURCE_PERMISSIONS)
+        if(NOT IS_SYMLINK ${source_file} AND WIN32)
```

Comment:
The condition looks incorrect. The file copy wouldn't be executed on non-Windows anymore?

Reply:
@seanyen Please double check the modified condition: 5895a4b6b6f994d34adaf90477a53522e51c955d.

Reply:
Thanks for the patch! I confirmed the modified version fixes the bug as well.